SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorLayoutDetails.h

```diff
@@ -11,7 +11,12 @@ DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKEditorLayoutDetails, Log, All);
 class FSpatialGDKEditorLayoutDetails : public IDetailCustomization
 {
 private:
-	FReply ClickedOnButton();
+	bool TryConstructMobileCommandLineArgumentsFile(FString& CommandLineArgsFile);
+	bool TryPushCommandLineArgs(const FString Executable, const FString ExeArguments, const FString CommandLineArgsFile);
+
+	FReply GenerateDevAuthToken();
+	FReply PrepareIOSApplication();
+	FReply PrepareAndroidApplication();
```

Comment:
Would be good to get a comment saying what prepare means.

Reply:
renamed the method